.github/workflows/npm-publish.yml

```diff
@@ -4,48 +4,37 @@
 name: Build & Deploy to NPM
 
 on:
-  push:    
-    tags:        
-      - '*.*.*'
-    
+  push:
+    tags:
+      - "*.*.*"
 
 jobs:
-
   build:
     runs-on: macos-latest
-    
+
     steps:
       - name: Actions checkout
         uses: actions/checkout@v3
-      
+
       - name: Setup node
         uses: actions/setup-node@v3
         with:
           node-version: latest
           registry-url: https://registry.npmjs.org/
-          
+
       - name: Setup Deno
         uses: denoland/setup-deno@v1
         with:
           deno-version: v1.x
 
       - name: Get the tag name
         id: get_tag_name
-        run: echo ::set-output name=TAG::${GITHUB_REF#refs/*/}
-        
+        run: echo "TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
```

Comment:
This is the only change being proposed in this PR. The rest of the changes are formatting corrections made by my IDE.